discovery-infra/test_infra/controllers/node_controllers/node_controller.py

```diff
@@ -3,12 +3,18 @@
 
 import libvirt
 
+from test_infra.helper_classes.config import BaseClusterConfig
+from test_infra.helper_classes.config.controller_config import ControllerConfig
 from test_infra.controllers.node_controllers.disk import Disk
 from test_infra.controllers.node_controllers.node import Node
 
 
 class NodeController(ABC):
 
+    def __init__(self, config: ControllerConfig, cluster_config: BaseClusterConfig):
+        self._config = config
+        self._cluster_config = cluster_config
```

Comment:
Why do the nodes needs the cluster config?

Reply:
It needs the cluster name and maybe more, for the extended controllers